configure

```diff
@@ -40,9 +44,7 @@ function print_help {
   echo "  --with-remotefs (default)         Include capabilities for remote file access."
   echo "  --no-remotefs                     Disable remote file access."
   echo
-  echo "  --python2.7                       Use Python 2.7 (default)."
-  echo "  --python3.5                       Use Python 3.5, default is Python 2.7."
-  echo "  --python3.6                       Use Python 3.6, default is Python 2.7."
+  echo "  --virtualenv                      Path to virtualenv executable (otherwise looks in \$PATH)."
```

Comment:
I think this might be the first parameter in the list that takes an argument. This is not indicated on the left column. There are many styles for this, but I think the main ones are `--virtualenv ARG/arg/<arg>`. For instance, look at clang/vim/python/rsync and you will see all three represented. Then there's the question what `ARG` should be, which perhaps should be `VIRTUALENV_EXE` (for instance `virtualenv --help` uses `PYTHON_EXE`).

Reply:
Thanks, good call. I'll try to mirror other scripts that do similar argument-taking.